packages/common/http/src/params.ts

```diff
@@ -290,13 +291,13 @@ export class HttpParams {
           case 'a':
           case 's':
             const base = (update.op === 'a' ? this.map!.get(update.param) : undefined) || [];
-            base.push(update.value!);
+            base.push(`${update.value!}`);
```

Comment:
nit: I think it'd be great to have a function call that would convert a value to a string (here and in the `indexOf` call below):

```suggestion
            base.push(valueToString(update.value));
```